website/content/docs/commands/daemon/add-token.mdx

```diff
@@ -17,6 +17,15 @@ If you authenticate to multiple Boundary instances, the client cache stores mult
 
 By adding auth tokens to your client cache, you can select which specific Boundary instance you want to search.
 
+<Note>
+
+The `search` operation only displays the resources that you have read permissions to view.
```

Comment:
For add-token in the client daemo, I'm not sure if we want to call out that you specifically need the `read` or `read:self` grant on `auth-token` to successfully add a token since it's possible to authenticate without actually being able to read the token in boundary which has tripped me up before in testing

Reply:
Thank you @ZedLi ! That does seem important. I modified the note on that page to read:

> The `search` operation only displays the resources that you have permissions to view. You must have the `read` or `read:self` grant on the auth token to successfully add it. If you are logged in as a user who does not have the permissions to view a resource, it does not display in the list of results.

Does it sound OK?